irohad/model/execution/impl/command_executor.cpp

```diff
@@ -73,13 +72,34 @@ namespace iroha {
     bool AppendRoleExecutor::hasPermissions(const Command &command,
                                             ametsuchi::WsvQuery &queries,
                                             const Account &creator) {
+      creator_ = creator;
       return checkAccountRolePermission(
           creator.account_id, queries, can_append_role);
     }
 
     bool AppendRoleExecutor::isValid(const Command &command,
                                      ametsuchi::WsvQuery &queries) {
-      // TODO 26/09/17 grimadas: check. No additional checks required
+      auto creator = creator_;
```

Comment:
Same here

Reply:
This is not even needed, since you have a private field in the same class

Reply:
Hm, reasonable